### ISSUE TITLE
Separate tasks for "seed" and "development" data

### ISSUE THREAD

**Issue by joeroe** [xronos-ch/xronos.rails#324]
According to Rails best practices, db:seed should only include data actually needed in production, see e.g.:

https://ninjadevel.com/seeding-database-ruby-on-rails/

**Comment by joeroe** [comment]
The only non-production data we are currently seeding is 100 c14s with FactoryBot (and associated records). With #456 we have rake tasks that can import realistic radiocarbon data, so this is no longer needed (if it ever was).

**Comment by MartinHinz** [comment]
I agree. They have been useful in development, but if more realistic data can replace them, I do not see any reason to keep the seeding in its current form.
